static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.ts

```diff
@@ -48,7 +50,19 @@ const fillAdvertSlots = async (): Promise<void> => {
 		.filter(
 			(adSlot) => !(isDCRMobile && adSlot.id === 'dfp-ad--top-above-nav'),
 		)
-		.map((adSlot) => new Advert(adSlot));
+		.map((adSlot) => {
+			let additionalSizes = {};
+
+			if (
+				window.guardian.config.page.contentType === 'Gallery' &&
+				adSlot.dataset.name?.includes('inline')
+			) {
+				additionalSizes = {
+					desktop: [adSizes.billboard, createAdSize(900, 250)],
```

Comment:
Removed from https://github.com/guardian/frontend/pull/25318/files#diff-bd054de0f93b3c5974cc80b7f32c6ef80f30ad70bc5648cce21983b7a29ba165L18, now needs to be added at advert initilization